scripts/preserve-data.js

```diff
@@ -0,0 +1,23 @@
+const contactForm = document.getElementById('contactForm');
+
+const data = {
+  name: String,
+  email: String,
+  message: String,
+};
```

Comment:
- [ ] **[OPTIONAL]:** Please avoid using `String` for each key of your data object. This isn't Type checking your data, instead, you're assigning the JavaScript's class String to each key inside the object. I think a better approach could be to define default values instead! 🤓 🔧 

```suggestion
const data = {
  name: '',
  email: '',
  message: '',
};
```